src/cloud/components/RateLimitAlert.tsx

```diff
@@ -110,6 +116,10 @@ const RateLimitAlert: FC<Props> = ({alertOnly, className, location}) => {
     ? IconFont.AlertTriangle
     : IconFont.Cloud
 
+  if (isFlagEnabled('multiOrg') && !multiOrgFlagForceShow) {
```

Comment:
if the `multiOrg` flag is ON, we're hiding the component everywhere unless `multiOrgFlagForceShow` prop is passed as true.

I checked that the RateLimitAlert is only used in page headers except on dashboard page. Where it's configured to show as an alert.